src/backend/tools.ts

```diff
@@ -43,7 +48,36 @@ export const DXVK = {
       },
       {
         name: 'dxvk',
-        url: 'https://api.github.com/repos/doitsujin/dxvk/releases/latest',
+        url: () => {
+          if (any_gpu_supports_version([1, 3, 0])) {
+            const instance_version = get_vulkan_instance_version()
+            if (
+              instance_version &&
+              semverLt(instance_version.join('.'), '1.3.0')
+            ) {
+              // FIXME: How does the instance version matter? Even with 1.2, newer DXVK seems to work fine
+              logWarning(
+                'Vulkan 1.3 is supported by GPUs in this system, but instance version is outdated',
+                LogPrefix.DXVKInstaller
+              )
+            }
+            return 'https://api.github.com/repos/doitsujin/dxvk/releases/latest'
+          }
+          if (any_gpu_supports_version([1, 2, 0])) {
+            logInfo(
+              'The GPU(s) in this system only support Vulkan 1.2, falling back to DXVK 1.10.3',
+              LogPrefix.DXVKInstaller
+            )
+            return 'https://api.github.com/repos/doitsujin/dxvk/releases/tags/v1.10.3'
+          }
+          logWarning(
+            'No GPU with Vulkan 1.2 support found, DXVK will not work',
```

Comment:
this should be changed to vulkan 1.1
https://github.com/doitsujin/dxvk/wiki/Driver-support#dxvk-1103